pkg/archive/diff.go

```diff
@@ -116,9 +119,23 @@ func UnpackLayer(dest string, layer Reader) (size int64, err error) {
 
 		if strings.HasPrefix(base, ".wh.") {
 			originalBase := base[len(".wh."):]
-			originalPath := filepath.Join(filepath.Dir(path), originalBase)
-			if err := os.RemoveAll(originalPath); err != nil {
-				return 0, err
+			dir := filepath.Dir(path)
+			if originalBase == ".wh..opq" {
```

Comment:
doesn't this name need an extra `.wh..`? Can we extract this to constants, so people know what they are?